privaterelay/settings.py

```diff
@@ -660,62 +651,44 @@ def _get_initial_middleware() -> list[str]:
     "usd": {
         "en": {
             "monthly": {
-                "id": "price_1LXUenJNcmPzuWtRw3rhjQNP",
+                "id": "TBD",
                 "price": "$?.??",
             },
             "yearly": {
-                "id": "price_1LXUhcJNcmPzuWtRHUFHVk12",
+                "id": "TBD",
                 "price": "$?.??",
             },
         },
```

Comment:
```suggestion
        },
        "ca": {
            "monthly": {
                "id": "price_1Li0w8JNcmPzuWtR2rGU80P3",
                "price": "$4.99",
            },
            "yearly": {
                "id": "price_1Li15WJNcmPzuWtRIh0F4VwP",
                "price": "$3.99",
            }
        },
```

Reply:
Hmm, I just mapped `ca` directly to the US entry, since [that's what the current mapping does](https://github.com/mozilla/fx-private-relay/blob/1a624ea0c303cc63890a511bad4cc8f362f8bf76/privaterelay/settings.py#L431-L433) and they're explicitly the same plan, but I might have misunderstood when to give items their own entry in the PLAN_ID_MATRIX and when to map them to existing entries in PLAN_COUNTRY_LANG_MAPPING?

Reply:
No that seems fine. Let's go with it.